packages/server/api/src/app/ai/chat/ai-mcp-chat.controller.ts

```diff
@@ -108,7 +108,7 @@ export const aiMCPChatController: FastifyPluginAsyncTypebox = async (app) => {
       content: request.body.message,
     });
 
-    const { mcpClients, tools } = await getMCPTools(
+    const { mcpClients, tools, isAwsCostMcpAvailable } = await getMCPTools(
```

Comment:
this should not be in here, do it like with the hasToolProvider below. 

eg. 

isAwsCostMcpAvailable = hasToolProvider(filteredTools, 'costAnalysisTools') || hasToolProvider(filteredTools, 'costExplorerTools')